tests/cases/fourslash/annotateWithTypeFromJSDoc13.ts

```diff
@@ -9,6 +9,6 @@ verify.codeFix({
     newFileContent:
 `class C {
     /** @return {number} */
-    get c(): number { return 12; }
+    get c(): number { return 12 }
```

Comment:
Why did we drop the semicolon?

Reply:
There's no semicolon in the original. Previously we would reformat the body, but now we just add a type annotation.